tests/common/devices/multi_asic.py

```diff
@@ -82,7 +82,8 @@ def critical_services_tracking_list(self):
                 if config_facts['FEATURE'][service]['state'] == "disabled":
                     self.sonichost.DEFAULT_ASIC_SERVICES.remove(service)
         for asic in active_asics:
-            service_list += asic.get_critical_services()
+            if asic:
+                service_list += asic.get_critical_services()
```

Comment:
why we need this active_Asic should only have fabric asics that are detected . see above logic